src/client/core/utils/dom.js

```diff
@@ -443,3 +443,9 @@ export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
 
     return null;
 }
+
+export function isEditableFormElement (element) {
+    var tag = getTagName(element);
+
+    return tag === 'select' || tag === 'input' || tag === 'textarea';
```

Comment:
it's better to use Hammerhead utils: [one](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/dom.js#L599), [two](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/dom.js#L431)